spec/unit/lib/temporal/workflow/state_manager_spec.rb

```diff
@@ -36,4 +38,30 @@ class MyWorkflow < Temporal::Workflow; end
       end
     end
   end
-end
\ No newline at end of file
+
+  describe '#search_attributes' do
```

Comment:
Can you add a test when no search attributes are set to make sure it returns `{}` ?